source/features/restore-file.tsx

```diff
@@ -42,23 +42,34 @@ async function getFile(filePath: string): Promise<{isTruncated: boolean; text: s
 	return repository.file;
 }
 
-async function dropFile(progress: (message: string) => void, menuItem: Element, filePath: string): Promise<void> {
+async function discardChanges(progress: (message: string) => void, filePath: string): Promise<void> {
 	const file = await getFile(filePath);
 
-	if (!file) {
-		// The file was created by this PR.
-		// This code won’t be reached if `highlight-deleted-and-added-files-in-diffs` works.
-		throw new Error('Nothing to drop. Delete file instead');
```

Comment:
This is now possible. We had only dropped this part because we used to load the native "delete" page and act on that. Now it's much easier via API

![2](https://user-images.githubusercontent.com/1402241/236630751-e3ec2ded-99c3-4386-bbd1-e2eb9077aef9.gif)